odb_extractor_atoms.py

```diff
@@ -346,7 +348,7 @@ def seqxlsx(sequence, comment='', path=''):
         if 'GMOS' in inst:
             coadds = '1'
             # convert wavelength to microns
-#             wavelength = '{:5.3f}'.format(float(wavelength) / 1000.)
+    #             wavelength = '{:5.3f}'.format(float(wavelength) / 1000.)
```

Comment:
As a note, `format` is pretty outdated. We should be using string interpolation. (This is a general comment and not to anyone in specific.)

```
print(f'{wavelength:5.3f}')
```
or even:
```
print(f'{float(wavelength)/1000:5.3f}')
```